game-core/src/main/java/games/strategy/triplea/ui/display/ITripleADisplay.java

```diff
@@ -37,71 +37,65 @@ void reportMessageToPlayers(final Collection<PlayerID> playersToSendTo,
    *        - a unique id for the battle
    * @param location
    *        - where the battle occurs
-   * @param battleTitle
-   *        - the title of the battle
    * @param attackingUnits
    *        - attacking units
    * @param defendingUnits
    *        - defending units
    * @param killedUnits
    *        - killed units
-   * @param dependentUnits
-   *        - unit dependencies, maps Unit->Collection of units
    * @param attacker
    *        - PlayerID of attacker
    * @param defender
    *        - PlayerID of defender
    */
-  void showBattle(GUID battleId, Territory location, String battleTitle, Collection<Unit> attackingUnits,
+  void showBattle(GUID battleId, Territory location, Collection<Unit> attackingUnits,
```

Comment:
`ITripleADisplay` is a remote interface.  I don't think you can change the signatures of the methods in this interface without breaking RMI compatibility.  I tried running a network game with a 3635 client, and I'm getting a ton of RMI-related errors, the first of which is

```
error in call:Remote method call, method name:showBattle remote name:games.strategy.engine.framework.AbstractGame.DISPLAY_CHANNEL
java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeSingle(EndPoint.java:157)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeMultiple(EndPoint.java:140)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeLocal(EndPoint.java:126)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.lambda$2(UnifiedMessenger.java:279)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Reply:
you're right.. that's why those params are all unused.